qiskit/circuit/quantumcircuit.py

```diff
@@ -1445,60 +1417,32 @@ def qasm(self, formatted=False, filename=None, encoding=None):
             if instruction.name == "measure":
                 qubit = qargs[0]
                 clbit = cargs[0]
-                string_temp += "{} {} -> {};\n".format(
+                string_temp += "{} {} -> {};\n".forma(
```

Comment:
This typo means that this code is not being covered in the tests. 
```suggestion
                string_temp += "{} {} -> {};\n".format(
```
Maybe add a test for it?

Reply:
Thanks for catching and fixing this. I have not idea why my VSCode didn't complain.